.github/workflows/ci-example.yml

```diff
@@ -0,0 +1,41 @@
+name: CI_Example
+
+on:
+  pull_request:
+    branches:
+      ci-add-examples-test-211
+
+jobs:
+  examples-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest] #[macos-latest, ubuntu-latest]
+        python-version: [3.7,]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[match-py-ver] --no-cache-dir
+      - name: Test with pytest
+        run: |
+          # make sure docker exist
+          docker ps
+          jina check
+          docker build  -f Dockerfiles/pip.Dockerfile -t jinaai/jina:test-pip .
```

Comment:
u do not need this one

Reply:
Unless we test an example running based on a jina docker image but not the case now